packages/core/test-helpers/core-test-helpers-deprecations-getters/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/core-test-helpers-deprecations-getters
+
+Empty package generated by @kbn/generate
```

Comment:
Would you mind adding a description for this package? 😇

Reply:
Good catch! I threw up this PR as a POC and didn't think to come back to the README.